dogs.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles/stylesheet.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Irish+Grover&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/d04e98713e.js" crossorigin="anonymous"></script>
+    <title>AdoptADog Dogs</title>
+    <meta name="author" content="Valerie V. Mehring">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+      <div class="cart"><img src="images/cart-icon.svg"> </div>
+      <p>$246.90</p>
+      <a href="checkout.html" class="button">checkout</a>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><i class="fas fa-paw"></i></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><i class="fas fa-paw"></i></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h3>Here are all of our dogs!</h3>
+      <figure>
+        <a href="#"><img src="images/murphy-card.jpg" alt="An adult hound mix named Murphy"></a>
```

Comment:
I would also recommend you change your dog tiles from being wrapped in a figure to a div. You'll then want to create a class to style these dog tiles. Next, you'll want to create another div that wraps all of your dog tile divs. You'll then want to create a class (maybe dog-tiles) and use css to create the grid. I would suggest using flex. Checkout this resource and see if it helps. You don't need to worry about including the @media queries quite yet. Just see if you can replicate what the article is doing. [https://css-tricks.com/dont-overthink-flexbox-grids/](https://css-tricks.com/dont-overthink-flexbox-grids/)